app/pathogen/arbovirus/dashboard/(visualizations)/median-seroprevalence-by-who-region-and-age-group-table.tsx

```diff
@@ -169,9 +170,22 @@ export const MedianSeroprevalenceByWhoRegionAndAgeGroupTable = () => {
   return (
     <div className="p-2">
       <div className="flex justify-between mb-2 ignore-for-visualization-download">
-        <h2 className="text-center">
-          {convertArboSFtoArbo(currentlySelectedArbovirus)}
-        </h2>
+        <DropdownMenu>
+          <DropdownMenuTrigger asChild>
+            <Button variant="outline" size="sm" className="mx-2 whitespace-nowrap text-white ignore-for-visualization-download">
```

Comment:
Yeah this looks pretty bad in the downloaded file so I excluded it. The downloaded file used to say which arbovirus it was for but now it doesn't anymore. I'm going to sleep on this, I think there's probably a better solution hiding somewhere.